### PR TITLE
Changes ahelp language to be more robust.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Changes ahelp response language to be more robust.

When an admin marks your ticket as an IC issue:
```
- Your AdminHelp has been marked as IC issue! -
An admin has carefully looked at the issue that you submitted and has determined that the reported incident is valid.
Some actions by users, while appearing malicious, can actually be legitimate due to their status or given situation in the round.
Please do not be discouraged from reporting similar instances in the future. While not all actions are invalid, not all actions are valid.
```

When an admin marks your ticket as unhelpful:
```
Your admin help was rejected. The adminhelp verb has been returned to you so that you may submit another ticket.
Please try to be calm, clear, and descriptive in admin helps. Assume that the admin has not seen any related events, and clearly state the character names or byond keys of anybody you are reporting.
```

When an admin starts to handle your ticket:
```
Your ticket is now being handled by an admin. Please wait while they type their response and gather relevant information.
```

## Why It's Good For The Game

The current language is really weak and doesn't really give anything for the person to chew on when it comes to learning the game. For the IC issue response, it assumes too much about why you're ahelping by stating "Losing is a part of the game.", which can cause the undesired effect of the admin not really knowing about the situation as not everyone who ahelps does it because they die or get caught. It also doesn't really explain that much for a button that is so commonly used, so that is also fixed.

It also re-words some of the responses to seem less direct and more friendly. For example, It doesn't say "do not assume that the admin knows everything" but says "assume the admin knows nothing". Language has a huge impact on mood and mannerisms which is why you never have tech support companies say "Calm and be patient before you speak to our operators." but rather "Please speak in a calm, clear manner when describing the problem."

Citation: Went to a filthy ~~LIBERAL SOCIALIST~~ University where they ~~CONDITIONED~~ taught me how to use language while getting my Computer Science certification.

## Changelog
:cl: BurgerBB
tweak: Tweaks ahelp preset responses to be more friendly.
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -387,7 +388,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(state != AHELP_ACTIVE)
 		return
 
-	var/msg = "<span class ='adminhelp'>Your ticket is now being handled by an admin. Please be patient.</span>"
+	var/msg = "<span class ='adminhelp'>Your ticket is now being handled by an admin. Please wait while they type their response and gather relevant information.</span>"
```
Comment (deathride58) [inline code/modules/admin/verbs/adminhelp.dm:391]: "Please be patient" is a lot more to-the-point and doesn't imply that the admin is going to be immediately typing up a response to an ahelp. The only thing this will do is make players more prone to being impatient when admins smack that "handle" button

File: code/modules/admin/verbs/adminhelp.dm [hunk 2]
```diff
@@ -369,9 +369,10 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(state != AHELP_ACTIVE)
 		return
 
-	var/msg = "<font color='red' size='4'><b>- AdminHelp marked as IC issue! -</b></font><br>"
-	msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
-	msg += "<font color='red'>It is also possible that your ahelp is unable to be answered properly, due to events occurring in the round.</font>"
+	var/msg = "<font color='red' size='4'><b>- Your AdminHelp has been marked as IC issue! -</b></font><br>"
+	msg += "<font color='red'><b>An admin has carefully looked at the submitted issue and has determined that the reported incident is valid.</b></font><br>"
+	msg += "<font color='red'>Some actions by users, while appearing malicious, can actually be legitimate due to their status or given situation in the round.</font>"
+	msg += "<font color='red'>Please do not be discouraged from reporting similar instances in the future. While not all actions are invalid, not all actions are valid.</font>"
```
Comment (deathride58) [inline code/modules/admin/verbs/adminhelp.dm:375]: Lol, imagine unironically turning the dedicated "grow a pair, dude" button into an overly patronizing "uwu its valid uwu uwu uwu" button
The "Resolve" and "Close" buttons are for ahelps about IC issues that don't exactly warrant telling someone to grow a pair. The IC issue button is for when someone is extra salty about dying via valid cause or otherwise being inconvenienced.

File: code/modules/admin/verbs/adminhelp.dm [hunk 3]
```diff
@@ -354,8 +354,8 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		SEND_SOUND(initiator, sound('sound/effects/adminhelp.ogg'))
 
 		to_chat(initiator, "<font color='red' size='4'><b>- AdminHelp Rejected! -</b></font>")
-		to_chat(initiator, "<font color='red'><b>Your admin help was rejected.</b> The adminhelp verb has been returned to you so that you may try again.</font>")
-		to_chat(initiator, "Please try to be calm, clear, and descriptive in admin helps, do not assume the admin has seen any related events, and clearly state the names of anybody you are reporting.")
+		to_chat(initiator, "<font color='red'><b>Your admin help was rejected.</b> The adminhelp verb has been returned to you so that you may submit another ticket.</font>")
+		to_chat(initiator, "Please try to be calm, clear, and descriptive in admin helps. Assume that the admin has not seen any related events, and clearly state the character names or byond keys of anybody you are reporting.")
```
Comment (deathride58) [inline code/modules/admin/verbs/adminhelp.dm:358]: This is literally the only change in this entire PR that'd actually be worth having.

**Review by deathride58 — COMMENTED** [review]
*yawn
Another day, another PR trying to make changes that only very specific types of people actually want.

File: code/modules/admin/verbs/adminhelp.dm (hunk 2 above)
Comment (BurgerLUA) [reply code/modules/admin/verbs/adminhelp.dm:375]: imagine having this shit-tier attitude. there is literally no reason to get this hostile over a pr lmao but sure.

The grow of pair button is no longer used as a grow a pair button but a quick and easy way of telling someone it's an IC issue regardless of how stupid the ahelp is. In 100% of instances where I ahelped shit that looked like bad behavior but I wasn't actually affected by the actions of the person, the button was used and there is nothing wrong with that. Literally everyone I talked to said the same thing and it's not because they are a part of the group of a "certain type of people" and the admins I talked to say they don't use it to tell people to grow a pair but to tell them that it's an IC issue without specifically saying "yeah that dude is an antag." or leaking any information about the round.

It's a useful button for admins dealing with a lot of shit and there is literally nothing wrong with that. Everyone knows that its the purpose. Admins don't have time to tell everyone in someone who died in a random explosion to kill one target that it was valid. Admins don't have time to tell everyone affected in a plasma flood 5 minutes into the shift that it's valid. If admins want to tell someone to grow a pair they literally use the "man up button" but I guess we're pretending that doesn't exist?

This is especially useful for new players and salty players. As I explained in the staff forum it's super useful for salty players to read and understand because it gives them more to chew on. It explains the purpose of the button and tells them to relax without specifically telling them to relax instead of just the "dying is a part of the game" which literally everyone and their dog knows.

File: code/modules/admin/verbs/adminhelp.dm (hunk 1 above)
Comment (BurgerLUA) [reply code/modules/admin/verbs/adminhelp.dm:391]: Justin, one of your better admins, requested this feature by the way. They think it's useful so who am I to argue?

**Comment by Poojawa** [comment]
I made the marked as IC button vague because being too wordy isn't required. All of these suggested changes are prattling on about what's already said. Considering the default marked as IC message is 'Losing is part of the game, LOL, eat shit' the one currently in gets to the point without being offhandedly dismissive.

**Comment by CozmoB** [comment]
All I want is a name of the admin hitting the "IC issue button". I was to be able to reference rulings between admins as currently there is a major problem of accountability and consensus.

**Comment by silicons** [comment]
since none of the headmins have told me this is good and i quite agree with the one that says it's too wordy i'm gonna go with nah.
granted ic issue button is lazy but /shrug

**Comment by BurgerLUA** [comment]
Then lets hope the admins rethink their policy with dealing with ahelps because I've been told by head admins that it's meant to be used in special circumstances yet in my experience and other people's experience, it's not.

**Comment by ShadeAware** [comment]
Holy jeez this was a yikes, do you people treat all your contributors like this @kevinz000 ?

**Comment by BobJogaIsForkinBanned** [comment]
Holy sht citadel. Is this really how you treat contributors?

**Comment by CozmoB** [comment]
Love getting updates on closed PR's 8 months after the fact to bitch at server admins rather than take it to dm's or a discord.

**Comment by ghost** [comment]
And people wonder why i hate Citadel when Contributors that mean well get treated this disgustingly
